src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -276,6 +276,30 @@ cdef class Buffer:
         hb_buffer_set_message_func(self._hb_buffer, msgcallback, <void*>callback, NULL)
 
 
+cdef class Blob:
+    cdef hb_blob_t* _hb_blob
+    cdef object _data
+
+    def __cinit__(self, bytes data):
+        if data is not None:
+            self._data = data
+            self._hb_blob = hb_blob_create(
+                data, len(data), HB_MEMORY_MODE_READONLY, NULL, NULL)
+        else:
+            self._hb_blob = hb_blob_get_empty()
+
+    @classmethod
+    def from_file_path(cls, filename: Union[str, Path]):
+        cdef bytes packed = str(filename).encode()
```

Comment:
I believe the correct way here would be to call os.fsencode() here, works on both str and PathLike objects and respects the filesystem encoding

Reply:
Done.